README.md

```diff
@@ -45,8 +48,8 @@ check out the [pillar documentation](https://github.com/comeara/pillar#migration
 The `cassandra.url` has to follow the format `cassandra://<host>:<port>/<keyspace>?host=<host>&host=<host>`, e.g. it would be
 `cassandra.url="cassandra://192.168.0.10:9042/my_keyspace?host=192.168.0.11&host=192.168.0.12"`.
 
-The `pillarReplicationStrategyConfigKey` is optional, the default value is `SimpleStrategy`. 
-The `pillarReplicationFactorConfigKey` is optional, the default value is `3`. 
+The `pillarReplicationStrategyConfigKey` is optional, the default value is `SimpleStrategy`.
+The `pillarReplicationFactorConfigKey` is optional, the default value is `3`.
 The `pillarDefaultConsistencyLevelConfigKey` is optional, the default value is `QUORUM`.
 
```

Comment:
Can you add a description for the usage of `pillarExtraMigrationsDirs`, when/how it is useful?